department.rb

```diff
@@ -0,0 +1,36 @@
+class Department
+  attr_accessor :name, :employees
+  def initialize(name)
+    @employees = []
+    @name = name
+  end
+
+  def add(employee)
+    employee.department = @name
+    @employees << employee
+  end
+
+  def salary
+    salaries = 0
+    @employees.each do |emp|
+      salaries += emp.salary
+    end
+    return salaries
+  end
+
+  def department_raise(dollars)
```

Comment:
I like the loop to accumulate the salary over which raises will be given, then a second loop to distribute.  It might be a bit cleaner to break this into multiple methods, but it's very good logic.